options.ml

```diff
@@ -78,6 +78,20 @@ module MaxArgs = Int
     let help = "set maximal number of function arguments"
   end)
 
+module Arrays = False
```

Comment:
Users should use options to *narrow* the language generated, not to widen it. So the default for anything that generates new language constructs should be `True`.

Reply:
Right.